README.md

```diff
@@ -43,6 +43,12 @@ After installing Zeal, you need to download docsets. Go to *File->Options->Docse
 
 To compile Zeal run `qmake` and then `make`. Linux users can install Zeal with `make install` command.
 
+### Instructions for OSX
```

Comment:
I think a [wiki](https://github.com/zealdocs/zeal/wiki) page would be a more appropriate place for such information.

Putting this here would assume some kind of support for OS X, while there's none.